sklearn/utils/sparsetools/setup.py

```diff
@@ -1,22 +1,15 @@
-import numpy
+# Remove in version 0.21
```

Comment:
got it. well, a bit annoying to require those placeholders but this seems to work.